pkg/scheduler/scheduler.go

```diff
@@ -499,7 +499,7 @@ func newPodInformer(cs clientset.Interface, resyncPeriod time.Duration) cache.Sh
 	tweakListOptions := func(options *metav1.ListOptions) {
 		options.FieldSelector = selector
 	}
-	informer := coreinformers.NewFilteredPodInformer(cs, metav1.NamespaceAll, resyncPeriod, cache.Indexers{}, tweakListOptions)
+	informer := coreinformers.NewFilteredPodInformer(cs, metav1.NamespaceAll, resyncPeriod, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc}, tweakListOptions)
```

Comment:
with #110663, we should be able to customize this indexer on plugin side. @xiaoanyunfei would you want to submit a PR on coscheduling plugin instead?

Reply:
Let me help with this, I'm working on coscheduling plugin. see https://github.com/kubernetes-sigs/scheduler-plugins/pull/654